articles/libraries/lock-ios/password-reset-ios.md

```diff
@@ -0,0 +1,52 @@
+---
+toc_title: Password Reset 
+description: All you need to know about password reset in iOS
+---
+
+# Lock iOS: Password Reset
+
+## Introduction
+You can allow the user to reset the password for any database connections. 
+We strongly suggest reading [this article](/articles/connections/database/password-strength.md) before implementing this.
```

Comment:
@lobo The valid URL is `/connections/database/password-strength` (we omit `www.auth0.com/docs` and add the rest; we also omit the `md` extension)